src/widgets/merch/ui/merch.module.scss

```diff
@@ -10,10 +10,11 @@
     }
   }
 
```

Comment:
The .info class has the property text-align: left; - this is the default value, I think it is redundant and can be removed

Reply:
Done